File: pranav/fib.c
unsigned long long fibonacci(unsigned long long n) {
    if (n == 0)
        return 0;
    else if (n == 1)
        return 1;
    else
        return fibonacci(n - 1) + fibonacci(n - 2);
}

int main() {
    int max_value = 40; // Adjust this value based on the desired intensity

    for (int i = 0; i < max_value; i++) {
        unsigned long long result = fibonacci(i);
    }

    return 0;
}

File: pranav/fib.rs
fn fibonacci(n: u64) -> u64 {
    match n {
        0 => 0,
        1 => 1,
        _ => fibonacci(n - 1) + fibonacci(n - 2),
    }
}

fn main() {
    let max_value = 40; // Adjust this value based on the desired intensity

    for i in 0..max_value {
        let result = fibonacci(i);
    }
}
